lib/encryption.js

```diff
@@ -123,7 +107,7 @@ var multiEncryptString = function (keystore, pwDerivedKey, msg, myPubKey, theirP
   encryptedSymKey = []
```

Comment:
This should be named `encryptedSymKeys` and `encSymKey` you can either name it `key` because of the context the reader can understand it's an encrypted key, or `encryptedSymKey` (singular). 
How it is now it's a little confusing because `encryptedSymKey` and `encSymKey` means basically the same thing.

Reply:
This is a good point, naming things is hard! :)

Reply:
@dekuyper This is a bit confusing. The reason it's called `encryptedSymKey` is that every entry in the list is an encrypted version of *the same* symmetric key. So there is only one symmetric key involved and it's encrypted to different counterparties. So I haven't been able to come up with a better name since `encryptedSymKeys` makes it sound like there are several symmetric keys involved.

Reply:
Also I'm a bit reluctant to change the name since it affects the serialized output.